server/remote_cache/content_addressable_storage_server/content_addressable_storage_server.go

```diff
@@ -1114,10 +1120,62 @@ func isComplete(children []*capb.DirectoryWithDigest) bool {
 	return true
 }
 
+// SpliceBlob is used to tell the server how it can assemble a blob from a list of CAS digests.
+// The server will verify the chunks assembled from the digests match the expected blob digest.
 func (s *ContentAddressableStorageServer) SpliceBlob(ctx context.Context, req *repb.SpliceBlobRequest) (*repb.SpliceBlobResponse, error) {
-	return nil, status.UnimplementedErrorf("SpliceBlob RPC is not currently implemented")
+	ctx, err := prefix.AttachUserPrefixToContext(ctx, s.env.GetAuthenticator())
+	if err != nil {
+		return nil, err
+	}
+
+	if efp := s.env.GetExperimentFlagProvider(); efp == nil || !efp.Boolean(ctx, experimentFlagChunkingEnabled, true) {
```

Comment:
Not really a comment on this PR, but this would read much nicer if `nil` implemented the flag provider interface as a noop that always returns default values.